### PR TITLE
Fix 7zip path for julia 1.3

### PR DESCRIPTION
**Pull request by oxinabox**
Since it got moved

### CODE REVIEW

File: src/BinDeps.jl [hunk 1]
```diff
@@ -114,7 +114,12 @@ if Sys.KERNEL == :FreeBSD
 end
 
 if Compat.Sys.iswindows()
-    const exe7z = joinpath(Compat.Sys.BINDIR, "7z.exe")
+    if isdefined(Base, :LIBEXECDIR)
+        const exe7z = joinpath(Sys.BINDIR, Base.LIBEXECDIR, "7z.exe")
+    else
+        const exe7z = joinpath(Sys.BINDIR, "7z.exe")
```
Comment (ararslan) [inline src/BinDeps.jl]:
```suggestion
        const exe7z = joinpath(Compat.Sys.BINDIR, "7z.exe")
```

**Comment by JackDunnNZ** [comment]
Any chance this can be merged? It seems to be breaking ScikitLearn.jl for me

**Comment by musm** [comment]
bump yes this is also causing failures

**Comment by musm** [comment]
Can we merge this ASAP and tag a release? It's causing any package using BinDeps failure on 1.3

**Comment by musm** [comment]
@ViralBShah I'm not sure who has permissions here, but this is a pretty important PR to get merge and make a release. I'ts has broken Blosc which has broken CMake and by extension HDF5.jl

**Comment by ViralBShah** [comment]
@musm gave you commit access here.

**Comment by musm** [comment]
Thank you very much!

**Comment by musm** [comment]
Any chance we can install https://github.com/JuliaRegistries/Registrator.jl here? It can only be performed by organization owners.

We'll also need to get in https://github.com/JuliaPackaging/BinDeps.jl/pull/408

**Comment by ViralBShah** [comment]
@musm I have temporarily invited you to be owner of JuliaPackaging, to help smoothen out things here.

**Pull request merged**